identity_storage/src/lib.rs

```diff
@@ -17,7 +17,14 @@
 pub mod key_id_storage;
 pub mod key_storage;
 pub mod storage;
+/// Wrapper around `SecretManager` that implements the storage interfaces.
```

Comment:
Modules are conventionally documented inside the `mod.rs` file with a `//!` comment.